.github/workflows/push.yml

```diff
@@ -13,8 +13,8 @@ jobs:
 
     - name: Run CI scripts
       run: |-
-        sudo make ci
+        ./scripts/ci-wrapper dev-v2.5
 
-    - uses: stefanzweifel/git-auto-commit-action@480e111bf8274f3bbd429aef51504f71c808e3e4
-      with:
-        commit_message: Generated changes
+    - name: Push charts and assets to the dev-v2.5 branch
+      run: |-
+        ./scripts/push-to-assets-branch origin dev-v2.5 && git push origin dev-v2.5
```

Comment:
```suggestion
        ./scripts/push-to-assets-branch origin dev-v2.5-source && git push origin dev-v2.5-source
```

Reply:
same, source is the branch that developer are working on. Target(asset) is dev-v2.5